packages/bpk-component-floating-notification/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "bpk-component-floating-notification",
+  "version": "1.0.0",
+  "description": "Backpack floating-notification component.",
+  "license": "Apache-2.0",
+  "private": true,
```

Comment:
So we can publish this can you remove this, I know this is the default of the boilerplate, so I will look at fixing that up to automatically remove it

Reply:
Removed!